pyglmnet/utils.py

```diff
@@ -130,3 +135,22 @@ def set_log_level(verbose):
             raise ValueError('verbose must be of a valid type')
         verbose = logging_types[verbose]
     logger.setLevel(verbose)
+
+
+def return_verbose_iterable(data):
+    """
+    Wrap an iterable object with tqdm. If tqdm is not available
+    or if we did not set the appropriate log level, then we
+    fall back to the classical method
+    :param data: it can be a list or range() object
+    :param verbose: the verbosity level
+    :return: wrapped object
+    """
+    wrapped_data = data
+    if logger.getEffectiveLevel() == logging.INFO:
+        if not has_tqdm:
+            logging.info(
+                "tqdm library not found. Falling back to non-interactive.")
```

Comment:
would this send out a message for every reg_lambda in the regularization path? We don't want to be too verbose either.

Reply:
Indeed this happens. I'll try to come up with something less annoying.

Reply:
sounds good!

Reply:
It should work better now.